tools/scripts/api-docs/pkg_tree.js

```diff
@@ -49,7 +49,7 @@ function main() {
 
 	dir = documentationPath();
 	if ( !exists( join( dir, 'package' ) ) ) {
-		mkdir( join( dir, 'package' ) );
+		mkdir( join( dir, 'package' ), { recursive: true } );
```

Comment:
Was this broken before? Or did something change such that providing this option become necessary?

Reply:
Yes, it was breaking trying to make `public/docs/api/v0.3.2/...` because `public/docs/api/v0.3.2/` didn't exist. for some reason this wasn't a problem for `public/docs/api/latest/...`, but I didn't look into why. If this is suspect, I'm happy to dig in to figure out why this was necessary and whether this is the best fix!